packages/nc-gui/pages/[projectType]/[projectId]/index.vue

```diff
@@ -491,10 +491,10 @@ onBeforeUnmount(reset)
       </a-layout-sider>
     </template>
 
-    <div :key="$route.fullPath.split('?')[0]">
+    <div>
       <LazyDashboardSettingsModal v-model="dialogOpen" :open-key="openDialogKey" />
 
-      <NuxtPage />
+      <NuxtPage :transition="false" :page-key="$route.params.projectId" />
```

Comment:
why transition false?

Reply:
It's leading to some issues if we close and open tabs too quickly, which is only reproducible in cypress. In cypress we already added some delay to take care of the issue, so we can revert if needed.

Reply:
Ok 👍